app/assets/javascripts/models/ui/spatial_type.js.coffee

```diff
@@ -7,25 +7,76 @@ ns.SpatialType = do (ko, $=jQuery) ->
     constructor: (@query) ->
       @icon = ko.observable('fa-crop')
       @name = ko.observable('Spatial')
+      @displaySpatial = ko.computed(read: @_getDisplaySpatialName, write: @_clearDisplaySpatialName, owner: this, deferEvaluation: true)
+      @querySpatial = ko.computed(read: @_getQuerySpatial, owner: this, deferEvaluation: true)
+      @mapControlTop = ko.computed(read: @_setMapControlPosition, owner: this)
```

Comment:
By convention, computed read methods start with `_compute` elsewhere in the code. Here you have three read methods, two of which start with `_get` and one of which starts with `_set`, which I think is confusing.